policyengine/countries/country.py

```diff
@@ -147,8 +147,16 @@ def household_reform(self, params=None):
         waterfall = household_waterfall_chart(
             baseline, reformed, self.results_config
         )
-        baseline.vary("employment_income", step=100)
-        reformed.vary("employment_income", step=100)
+        baseline.vary(
+            "employment_income",
+            step=100,
+            max=max(200000, baseline.calc("employment_income").sum() * 1.5),
```

Comment:
How do you mean?

Reply:
Oh I guess employment income is the same in `baseline` and `reformed`. If they differed, this would vary the models differently and the charts wouldn't align.

This won't create problems, but I think it'd be cleaner to make a variable `vary_max = max(200000, baseline.calc("employment_income").sum() * 1.5)` and pass that to both `baseline.vary` and `reformed.vary`.

Reply:
Yeah, that's better. Done!